src/modules/RootPart.rb

```diff
@@ -1104,6 +1104,29 @@ def MountFSTab(fstab, message)
         )
       end
 
+      # bind-mount /run into chroot (bsc#1152530)
+      if MountPartition("/run", "/run", "none", "bind") == nil
+        AddMountedPartition(
+          { :type => "mount", :device => "none", :mntpt => "/run" }
+        )
+      end
+
+      efivars_path = "/sys/firmware/efi/efivars"
```

Comment:
NP: This could be a const EFIVARS_PATH (and then freeze the string).

Reply:
That's just porting the patch.